includes/admin/class-rop-admin.php

```diff
@@ -1572,17 +1572,26 @@ public function rop_get_yoast_seo_active_status() {
 	public function rop_get_wpml_languages() {
 
 		if ( $this->rop_get_wpml_active_status() === false ) {
-					 return;
+			return;
 		}
 
 		$wpml_active_languages = apply_filters( 'wpml_active_languages', null, array('skip_missing' => 1) );
 
+		// Get TranslatePress publish plugin languages.
+		if ( empty( $wpml_active_languages ) && class_exists( 'TRP_Translate_Press' ) ) {
+			$trp_settings = TRP_Translate_Press::get_trp_instance()->get_component( 'settings' )->get_settings();
+			if ( $trp_settings ) {
+				$trp_languages         = TRP_Translate_Press::get_trp_instance()->get_component( 'languages' );
+				$publish_languages     = ! empty( $trp_settings['publish-languages'] ) ? $trp_settings['publish-languages'] : array();
+				$wpml_active_languages = $trp_languages->get_language_names( $publish_languages, 'english_name' );
```

Comment:
Here, we need to do a little debating. By using `english_name`, German language will appear as `German` and not `Deutsch`; by using `null`, we will have this based on documentation: `English_or_native_name parameter set to null means to obey the admin settings.`

@vytisbulkevicius I will let you decide what is best here. Having English names or to be the same as in WP settings: 
<img width="659" alt="Screenshot 2025-01-30 at 16 07 55" src="https://github.com/user-attachments/assets/9fe002db-64b8-4182-82d1-2d6622c129ff" />

Reply:
Fixed, Now, the native name is shown